skydb/skydb.go

```diff
@@ -156,7 +180,18 @@ func uploadData(c *client.Client, content []byte) (string, error) {
 		Mode:     skymodules.DefaultFilePerm,
 		Reader:   bytes.NewReader(content),
 	}
-	skylink, _, err := c.SkynetSkyfilePost(*sup)
+	var skylink string
+	var err error
+	// skyd might not be ready by the time we execute this, so we want to retry
+	// several times with increasing gaps in between.
+	for i := 0; i < 10; i++ {
```

Comment:
extract all of these magic numbers into constants `10` and `15`

Reply:
Removed.